src/PSDMatrices.jl

```diff
@@ -37,9 +37,9 @@ function det(M::PSDMatrix)
     return det(M.R)^2
 end
 
-function logdet(M::PSDMatrix)
+function logabsdet(M::PSDMatrix)
     confirm_factor_is_square(M)
-    return 2 * logdet(M.R)
+    return 2 * logabsdet(M.R)[1], 1.0
```

Comment:
Thanks! I appreciate your immediate update :) But I have to remain a little critical, unfortunately :(.

I personally found the `val, sgn = logabsdet(M.R); return 2*val, sgn^2` more readable and I think it is also more correct. For example, because the sign returned by `logabsdet` is usually an `Int64`, and the `Float64` here might confuse a user? 

Apologies if my previous comment might have sounded like I prefer returning 1 instead of squaring, it was meant as more of a question...

Reply:
I'll revert the last commit then :+1:

Reply:
Also feel free to suggest a change if you want to rename the variables, I don't have strong opinions on the underscores.

Reply:
I would've used `val(ue), sgn` instead of the underscores, but I don't have a strong opinion on the variable names. They have a lifespan of _one_ line anyway.